### PR TITLE
Make it runnable with python3

### PR DESCRIPTION
**Pull request by chengchingwen**

### CODE REVIEW

File: saved_networks/checkpoint [hunk 1]
```diff
@@ -1,4 +1,2 @@
-model_checkpoint_path: "bird-dqn-30000"
+model_checkpoint_path: "bird-dqn-10000"
 all_model_checkpoint_paths: "bird-dqn-10000"
-all_model_checkpoint_paths: "bird-dqn-20000"
-all_model_checkpoint_paths: "bird-dqn-30000"
```
Comment (yenchenlin1994) [inline saved_networks/checkpoint]: I don't think we need to change `saved_networks/checkpoint`

**Comment by chengchingwen** [comment]
yeah, I agree, It automatically modify the file and I accidentally commit it, I'll try to fix it

**Comment by chengchingwen** [comment]
I fix it

**Comment by yenchenlin1994** [comment]
Hello @chengchingwen , it works well for me.

However, it will add additional bracket "(" and ")" in `print`'s output when I run it in Python 2.7,
anyway you think can address it?

**Comment by chengchingwen** [comment]
I import print function from  __future__

**Comment by yenchenlin1994** [comment]
Cool!
Let's merge this.
Thanks a lot :smile:

**Comment by yenchenlin1994** [comment]
Hello @chengchingwen ,
just a reminder.

Since all these three commits are actually doing **one** thing - make it runnable with python3,
it will be more clear if you can squash them into **one** commit and it will be more clear when we review commit log in the future.

Still great thanks for your help :smile:

**Pull request merged**